services/distribution/src/main/resources/dgc/cclrules.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "identifier": "CCL-DE-0001",
```

Comment:
this is filtered because of older version, right?

Reply:
yes, Thats right. This is the test where it is confirmed  shouldCreateCorrectFileStructureForCommonCovidLogicRules()